jbmc/unit/Makefile

```diff
@@ -111,6 +108,7 @@ CPROVER_LIBS =../src/java_bytecode/java_bytecode$(LIBEXT) \
               $(CPROVER_DIR)/src/ansi-c/ansi-c$(LIBEXT) \
               $(CPROVER_DIR)/src/cpp/cpp$(LIBEXT) \
               $(CPROVER_DIR)/src/json/json$(LIBEXT) \
+	      $(CPROVER_DIR)/src/json_symtab_language/json_symtab_language$(LIBEXT) \
```

Comment:
This needs '-' rather than '_' in the directory name :-) but should be working once that's changed.

Reply:
Yes, my bad.